examples/flutter_gallery/lib/demo/video_demo.dart

```diff
@@ -97,6 +97,44 @@ class VideoCard extends StatelessWidget {
   }
 }
 
+class VideoPlayerLoading extends StatefulWidget {
+  final VideoPlayerController controller;
+
+  const VideoPlayerLoading(this.controller);
+
+  @override
+  _VideoPlayerLoadingState createState() => new _VideoPlayerLoadingState();
+}
+
+class _VideoPlayerLoadingState extends State<VideoPlayerLoading> {
+  bool initialized;
+
+  @override
+  void initState() {
+    super.initState();
+    initialized = widget.controller.value.initialized;
+    widget.controller.addListener(() {
+      if (!mounted) {
+        return;
+      }
+      if (initialized != widget.controller.value.initialized) {
+        setState(() {
+          initialized = widget.controller.value.initialized;
+        });
+      }
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final List<Widget> children = <Widget>[new VideoPlayer(widget.controller)];
```

Comment:
Perhaps just return the video player, if `initialized`?

Reply:
done